.github/actions/poetry_deps/action.yaml

```diff
@@ -16,7 +16,7 @@ runs:
         path: | 
           ~/.cache/pypoetry
           **/${{ inputs.poetry-dir }}/.venv
-        key: ${{ runner.os }}-poetry-${{ hashFiles('**/${{ inputs.poetry-dir }}/poetry.lock') }}
+        key: ${{ runner.os }}-poetry-${{ hashFiles('**/poetry.lock') }}
```

Comment:
not totally sure why, but including the `inputs.poetry-dir` in the glob match breaks the whole thing. I think this will technically hash both of the `poetry.lock` files in the repository, but I don't think that's a major problem

Reply:
super weird. and yeah, probably fine. it will result in a rebuild some time, but thats fine.